config/locales/it.yml

```diff
@@ -1035,10 +1573,12 @@ it:
     name: Nome
     name_on_card: Nome sulla carta
     name_or_sku: Nome o SKU (inserisci almeno i primi 4 caratteri del nome del prodotto)
+    negative_movement_absent_item: Impossibile creare movimento negativo per articoli
```

Comment:
what about `per articoli assenti in magazzino`?

Reply:
you are right! :) thank you. done!